### DISCUSSION TITLE
Is there Docs of flax version 0.5.1?

### DISCUSSION THREAD

**Discussion by wisdomGEsLA** [google/flax#2527] [Q&A] [answered]
I have to use flax.optim since it's used in a code written by others, but the latest Docs seems to delete them all, so is there a Docs of flax version 0.5.1?

**Comment by cgarciae** [comment, marked as answer]
We have docs for version `0.5.3` that contains the docs for `optim` here:

https://flax.readthedocs.io/en/v0.5.3/

You can switch version in the bottom left corner (its a bit hidden by the UI):

![Screenshot from 2022-10-11 11-54-20](https://user-images.githubusercontent.com/5862228/195153603-9e377b32-d8b1-48ea-866b-3d81cb877531.png)

**Reply by wisdomGEsLA** [reply]
I get your point, But it would be better if there's Docs have examples about flax.optim in chapters like GUIDED TOUR or Examples .Anyway, thank you for you answer!

**Reply by cgarciae** [reply]
@wisdomGEsLA we are actually going in the opposite direction 😅

https://github.com/google/flax/pull/2522

**Reply by 8bitmp3** [reply]
Thanks @wisdomGEsLA Let us know how it goes.
